### ISSUE TITLE
Should be able to manually restart an app when crashed

### ISSUE THREAD

**Issue by leonaves** [remy/nodemon#1426]
This is a duplicate of #764, which seems to be back, or still presenting in certain environments.

- `nodemon -v`: 1.18.4
- `node -v`: v10.10.0
- Operating system/terminal environment: alpine:3.8
- Command you ran: `nodemon script.js`

### Expected behaviour
Nodemon should restart when typing `rs` even after `[nodemon] app crashed - waiting for file changes before starting...`.

### Actual behaviour
Nodemon does not restart

### Steps to reproduce
```cli
docker run -it node:10.10.0-alpine /bin/ash
/ # cd /tmp
/tmp # yarn add nodemon
/tmp # echo 'console.log("works when not crashed");' > control.js
/tmp # yarn nodemon control.js
[nodemon] starting `node control.js`
works when not crashed
[nodemon] clean exit - waiting for changes before restart
rs
[nodemon] starting `node control.js`
works when not crashed
[nodemon] clean exit - waiting for changes before restart
^C
/tmp # echo 'console.log("fails when crashed";' > issue.js
/tmp # yarn nodemon issue.js
[nodemon] starting `node issue.js`
[nodemon] app crashed - waiting for file changes before starting...
rs
rs
rs
```

(The above is truncated, but provide the same commands and the problem is easily shown).

This isn't an issue running natively on MacOS. I would guess it was an issue with STDIN, what with it being in a container, however `rs` does restart when it isn't crashed, so I suspect it's something deeper.

---

If applicable, please append the `--dump` flag on your command and include the output here **ensuring to remove any sensitive/personal details or tokens**.

**Comment by leonaves** [comment]
Come on, I couldn't have made this easier to test. The commands are all there, at least acknowledge it.

**Comment by leonaves** [comment]
Could we not....
